WordPress/Classes/ViewRelated/Blog/BlogListViewController.m

```diff
@@ -31,6 +31,9 @@ typedef NS_ENUM(NSInteger, BlogListSections) {
 static CGFloat const BLVCHeaderViewLabelPadding = 10.0;
 static CGFloat const BLVCSiteRowHeight = 74.0;
 
+static int DisableAllMinSites = 10;
+static int DisableAllSitesThreshold = 6;
+static float DisableAllSitesInterval = 2.0f;
```

Comment:
Follow up... since `DisableAllSitesInterval` is used in a comparison with an `NSTimeInterval`, its type should probably also be an `NSTimeInterval` for consistency.